bucket/dreamset.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.4.13",
+    "description": "Dreamset is a settings editor for digital satellite receivers.",
+    "homepage": "http://www.dreamset-editor.com",
+    "license": "Unknown",
+    "architecture": {
+        "64bit": {
+            "url": "http://www.dreamset-editor.com/files/dreamset2413x64.zip",
+            "hash": "sha1:b24d4a3d0b1802b90ca6423d960c5b8cc367f6e2"
+        },
+        "32bit": {
+            "url": "http://www.dreamset-editor.com/files/dreamset2413.zip",
+            "hash": "sha1:522733f6a1103ef6dd412f981573d4706bcfa83f"
+        }
+    },
+    "bin": "dreamset.exe",
+    "shortcuts": [
+        [
+            "dreamset.exe",
+            "Dreamset"
+        ]
+    ],
+    "checkver": {
+        "url": "http://www.dreamset-editor.com/index.php",
+        "regex": "Dreamset ([\\d.]+) \\(32bit\\)"
```

Comment:
```suggestion
        "url": "https://www.dreamset-editor.com/dreamset-history",
        "regex": "-\\s+ver\\s+([\\d.]+)"
```

Reply:
Does the regex hold only the latest version is this would match all version numbers in the page?

Reply:
First match